lua/melange/build.lua

```diff
@@ -121,6 +121,33 @@ local function generate_iterm2(palette)
   return table.concat(template, '\n')
 end
 
+local function generate_windows_terminal_theme(variant, palette)
+  local template = [=[
+{
+"name": "melange $variant",
+"tab": 
+{
+  "background": "$bg",
+  "unfocusedBackground": null
+},
+"tabRow": 
+{
+  "background": "$black",
+  "unfocusedBackground": "$black"
+},
+"window": 
+{
+  "applicationTheme": "$variant"
+}
+}
+  ]=]
+
+  -- Append "FF" to color replacements to account for alpha
+  return template:gsub("$variant", variant)
+    :gsub("$black", palette.black .. "FF")
+    :gsub("$bg", palette.bg .. "FF")
```

Comment:
You can use `interpolate` here.

Reply:
I do apologize, using the interpolate doesn't seem to work since the theme is expecting an alpha. I figured the safest option was to just do it manually, since there were only a few replacements. 

Looking deeper into this, I believe the other options would be to modify the interpolation function (seems unsafe) or to copy the palette and modify the colors to append the alpha.

Forgive my ignorance if I'm not seeing this clearly though, as Lua is not one of the languages that I write in my day job.

Reply:
Yeah, sorry. I meant doing something like

```suggestion
  return interpolate(template, {
    variant = variant,
    black = palette.black .. "FF",
    bg = palette.bg .. "FF",
  })
```

Reply:
Nice solution. That is pretty nifty.